lib/phoenix/live_dashboard/components/table_component.ex

```diff
@@ -186,6 +204,21 @@ defmodule Phoenix.LiveDashboard.TableComponent do
         </div>
       </form>
 
+    <%= if @filter do %>
```

Comment:
Nitpick: indentation is off, can you please address or run the formatter in this file? Thanks!

```suggestion
     <%= if @filter do %>
```